### PR TITLE
fixed remove custome from nobs

### PR DESCRIPTION
**Pull request by darmhoo**

### CODE REVIEW

File: src/components/PaymentLog.vue [hunk 1]
```diff
@@ -170,7 +170,7 @@
                   v-model="salesLogForm.repayment_cycle_id"
                   v-validate="'required'"
                   @change="customDate($event)"
-                  :disabled="isCashNCarry"
+                  :disabled="isCashNCarry || lockRepaymentCycle"
```
Comment (teebarg) [inline src/components/PaymentLog.vue:173]: What does `lockRepaymentCycle` mean?

Comment (darmhoo) [reply src/components/PaymentLog.vue:173]: a prop to disable the dropdown if nobs is selected

File: src/components/PaymentLog.vue [hunk 2]
```diff
@@ -888,14 +889,20 @@ export default {
...
+      get(){
+        let newArray = []
       this.isAltaraPay
         ? (newArray = this.repaymentCyclesopt.filter(item => {
             return item.name !== "monthly"
           }))
         : (newArray = this.repaymentCyclesopt)
       return newArray
+      },
+      set(array){
```
[the remark this answers is not in the document]
Comment (darmhoo) [reply src/components/PaymentLog.vue:902]: it is an omission

File: src/components/PaymentLog.vue (hunk 1 above)
Comment (teebarg) [reply src/components/PaymentLog.vue:173]: What are we disabling? we want to use the existing calculator for monthly repayments

Comment (darmhoo) [reply src/components/PaymentLog.vue:173]: yes. when it is nobs, it should preselect bi-monthly and lock the dropdown, but if not nobs, it works normally

Comment (darmhoo) [reply src/components/PaymentLog.vue:173]: there in no monthly option for altara pay

Comment (teebarg) [reply src/components/PaymentLog.vue:173]: no we dont want to lock it, they should be able to select monthly as well and work as it is currently

**Pull request closed without merging**